extmod/modbluetooth.c

```diff
@@ -596,13 +612,17 @@ STATIC mp_obj_t bluetooth_ble_gap_scan(size_t n_args, const mp_obj_t *args) {
             }
         }
     }
+
+    bluetooth_handle_uninitialized();
```

Comment:
I don't think this is needed.  It currently raises `OSError(EIO)` if it's not active (on PYBD).